package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/helix-resolve-git-ref",
-  "version": "1.7.20",
+  "version": "1.7.20-hedy",
```

Comment:
We should probably revert this

Reply:
doesn't matter...semantic release will use a new version number anyways